src/Data/Rational/Base.agda

```diff
@@ -64,6 +64,9 @@ open ℚ public using ()
 mkℚ+ : ∀ n d → .{d≢0 : d ≢0} → .(Coprime n d) → ℚ
 mkℚ+ n (suc d) coprime = mkℚ (+ n) d coprime
 
+mkℚ*+ : ∀ n d → .{d≢0 : d ≢0} → .(Coprime (suc n) d) → ℚ
+mkℚ*+ n (suc d) coprime = mkℚ +[1+ n ] d coprime
```

Comment:
I'm probably being slow, but what's the advantage of this over `mkℚ`?

Reply:
Nothing really, but the same could be argued for `mkℚ+` defined one line above, so my reasoning was that if the latter is there, we could add other similar shorthands.